libraries/app/application.cpp

```diff
@@ -340,6 +340,9 @@ void application_impl::set_api_limit() {
 	if(_options->count("api-limit-get-key-references")){
 		_app_options.api_limit_get_key_references = _options->at("api-limit-get-key-references").as<uint64_t>();
 	}
+   if(_options->count("api-limit-get-htlc-by")) {
+      _app_options.api_limit_get_htlc_by = _options->at("api-limit-get-htlc-by").as<uint64_t>();
```

Comment:
I would have liked a more descriptive name for this, and have it cover both "to" and "from", but I could not think of one.